app/validators/blocks/list_collector_validator.py

```diff
@@ -0,0 +1,96 @@
+from app import error_messages
+from app.validators.blocks.block_validator import BlockValidator
+from app.validators.blocks.validate_collector_quesitons_mixin import (
+    ValidateCollectorQuestionsMixin,
+)
+
+
+class ListCollectorValidator(BlockValidator, ValidateCollectorQuestionsMixin):
+    def validate(self):
+        super(ListCollectorValidator, self).validate()
```

Comment:
These calls to `super` can be written:

```suggestion
        super().validate()
```

in Python 3